tests/common/main_test.go

```diff
@@ -22,6 +22,7 @@ import (
 )
 
 var testRunner = framework.UnitTestRunner
+var withAuth = func(userName, password string) config.ClientOption { return func(any) {} }
```

Comment:
ditto

Reply:
A default empty `withAuth` is needed to pass the local syntax check.

By default, `integration` and `e2e` build tag is not applied.

Reply:
FYI. https://github.com/etcd-io/etcd/pull/14628 

It should be more elegant!

Reply:
THe PR https://github.com/etcd-io/etcd/pull/14628 has been merged. Please remove `withAuth`, and rebase this PR.